perf/src/sigverify.rs

```diff
@@ -99,20 +103,32 @@ fn verify_packet(packet: &Packet) -> u8 {
         let sig_end = sig_start as usize + size_of::<Signature>();
 
         if pubkey_end >= packet.meta.size || sig_end >= packet.meta.size {
-            return 0;
+            packet.meta.discard = true;
         }
 
         let signature = Signature::new(&packet.data[sig_start..sig_end]);
+
         if !signature.verify(
             &packet.data[pubkey_start..pubkey_end],
             &packet.data[msg_start..msg_end],
         ) {
-            return 0;
+            // Check for tracer pubkey here
+            packet.meta.discard = true;
+            return;
+        }
+
+        if &packet.data[pubkey_start..pubkey_end] == TRACER_KEY.as_ref()
```

Comment:
I think that lazy static references can be slow, not sure this is good to do for every packet. I think a const or non-lazy static would be better

Reply:
Changed to initialize from `const` pubkey bytes